packages/frontend/src/ee/pages/AiAgents/AiAgentNewThreadPage.tsx

```diff
@@ -96,6 +105,14 @@ const AiAgentNewThreadPage = () => {
                         )}
                     </Stack>
 
+                    {verifiedQuestions && verifiedQuestions.length > 1 && (
```

Comment:
if the internal component takes care of the `verifiedQuestions` length, then this condition isn't necessary imo